### PR TITLE
Add CITATION.cff, tributors, zenodo.json

### PR DESCRIPTION
**Pull request by dcherian**
Following Earthcube review comments.

I used

https://citation-file-format.github.io/
https://con.github.io/tributors/docs/getting-started
https://github.com/castelao/inception

for guidance. In theory we could set up  a tributors github action but i'll leave that for later.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  orcid: https://orcid.org/0000-0003-1192-0403
+- affiliation: National Center for Atmospheric Research, USA
+  family-names: Kent
+  given-names: Julia
+- family-names: Magin
+  given-names: Justus
+- family-names: Thielen
+  given-names: Jon
+- family-names: Thyng
+  given-names: Kristen
```
Comment (dcherian) [inline CITATION.cff:37]:
```suggestion
  given-names: Kristen
  orcid: https://orcid.org/0000-0002-8746-614X
```

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  family-names: Bourgault
+  given-names: Pascal
+  orcid: https://orcid.org/0000-0003-1192-0403
+- affiliation: National Center for Atmospheric Research, USA
+  family-names: Kent
+  given-names: Julia
+- family-names: Magin
+  given-names: Justus
+- family-names: Thielen
+  given-names: Jon
```
Comment (dcherian) [inline CITATION.cff:35]: @jthielen can you provide your affiliation and orcid?

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  orcid: https://orcid.org/0000-0001-6849-3647
+- affiliation: Ouranos, Inc.
+  family-names: Bourgault
+  given-names: Pascal
+  orcid: https://orcid.org/0000-0003-1192-0403
+- affiliation: National Center for Atmospheric Research, USA
+  family-names: Kent
+  given-names: Julia
+- family-names: Magin
+  given-names: Justus
```
Comment (dcherian) [inline CITATION.cff:33]: @keewis can you provide an affiliation and orcid? Only if you want to...

File: CITATION.cff [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  family-names: Almansi
+  given-names: Mattia
+  orcid: https://orcid.org/0000-0001-6849-3647
+- affiliation: Ouranos, Inc.
+  family-names: Bourgault
+  given-names: Pascal
+  orcid: https://orcid.org/0000-0003-1192-0403
+- affiliation: National Center for Atmospheric Research, USA
+  family-names: Kent
+  given-names: Julia
```
Comment (dcherian) [inline CITATION.cff:31]: @jukent can you provide your orcid number?

File: CITATION.cff (hunk 3 above)
Comment (keewis) [reply CITATION.cff:33]: I don't have any of those, yet...

File: CITATION.cff (hunk 2 above)
Comment (jthielen) [reply CITATION.cff:35]: Hopefully this works! I'll have to remember to update this in a couple months when I change institutions.

```suggestion
  given-names: Jon
  orchid: https://orcid.org/0000-0002-5479-0189
  affiliation: Iowa State University, Ames, IA, USA
```

File: .tributors [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+        "bio": "Physical oceanography graduate turned climate science specialist and scientific developer.",
+        "orcid": "0000-0003-1192-0403",
+        "affiliation": "Ouranos Inc"
+    },
+    "keewis": {
+        "name": "Justus Magin"
+    },
+    "jukent": {
+        "name": "Julia Kent",
+        "affiliation": "National Center for Atmospheric Research"
```
Comment (dcherian) [inline .tributors:26]:
```suggestion
        "affiliation": "National Center for Atmospheric Research",
        "orcid": "0000-0002-5611-8986"
```

File: .zenodo.json [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+            "orcid": "0000-0001-6849-3647"
+        },
+        {
+            "name": "Pascal Bourgault",
+            "affiliation": "Ouranos Inc",
+            "orcid": "0000-0003-1192-0403"
+        },
+        {
+            "name": "Julia Kent",
+            "affiliation": "National Center for Atmospheric Research"
```
Comment (dcherian) [inline .zenodo.json:20]:
```suggestion
            "affiliation": "National Center for Atmospheric Research",
            "orcid": "0000-0002-5611-8986"
```

File: CITATION.cff [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
```
Comment (dcherian) [inline CITATION.cff:31]:
```suggestion
  given-names: Julia
  orcid: https://orcid.org/0000-0002-5611-8986
```

**Pull request merged**